### DISCUSSION TITLE
账户登录不上去，请帮助

### DISCUSSION THREAD

**Discussion by JXXL888** [community/community#38453] [Other Feature Feedback, Questions, & Ideas] [answered]
账户登录不上去，提示违规，不知道出了什么问题，请帮助。邮箱是：LJXZXL@gmail.com,  账户：LJXZXL88

**Comment by ws-jm** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!
My account flagged please suggest me
Hello, Mr. or Ms.
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support.
I don’t know why my account has been flagged.
